components/bl-dropzone/src/index.js

```diff
@@ -38,50 +36,44 @@ export default function DropzoneComponent({ component, eventHandlers }) {
         onChange={ updateFiles }
         value={ files }
         clickable={ clickable }
+        autoClean={ autoClean }
         onClean={ cleanButtonVisibility ? handleClean : false }
         accept={ acceptedFileTypes }
         maxFileSize={ maxFileSize }
         maxFiles={ maxFiles }
         label={ label }
-        minHeight={ ensureMeasure(minHeight) }
-        maxHeight={ ensureMeasure(maxHeight) }
+        minHeight={ normalizeDimensionValue(minHeight) }
         localization={ language }
-        view={ viewMode === 'unset' ? false : viewMode }
         footer={ footerVisibility }
         header={ headerVisibility }
         behaviour={ onDropBehaviour }
         color={ themeColor }
-        disableScroll={ disableScrollbar }
-        onDrop={ filesList => onDrop({ filesList }) }
-        onChangeView={ viewMode => onChangeView({ viewMode }) }
-      >
+        disableRipple={ disableRipple }
+        disabled={ disabled }>
+
         { files?.map(file => (
-          <FileItem
+          <FileMosaic
             { ...file }
             key={ file.id }
             onDelete={ handleDelete }
-            onSee={ handleSee }
-            onWatch={ handleWatch }
+            onSee={ preview ? handleSee : undefined }
+            onWatch={ preview ? handleWatch : undefined }
             alwaysActive={ activeItems }
             localization={ language }
             preview={ preview }
             info={ showInfoLayer }
-            hd
+            downloadUrl={ downloadButtonVisibility ? URL.createObjectURL(file.file) : undefined }
+            imageUrl={ file.type.match('image.*') && file.file } // fix fullscreen preview for not valid image files
+            videoUrl={ file.type.match('video.*') && file.file } // fix fullscreen preview for video files
           />
         )) }
       </Dropzone>
-      <FullScreenPreview
-        imgSource={ imageSrc }
-        openImage={ imageSrc }
-        onClose={ () => handleSee(undefined) }
-      />
-      <VideoPreview
-        videoSrc={ videoSrc }
-        openVideo={ videoSrc }
-        onClose={ () => handleWatch(undefined) }
-        controls
-        autoplay
-      />
+      <FullScreen open={ imageSrc !== undefined } onClose={ handleSee }>
```

Comment:
Wondering about why `imageSrc: false` is positive variant here.

Reply:
The `imageSrc` value cannot be `false`.
By default, it is `undefined`, when clicking the preview button, we set it to `imageSource` or `videoSource`, when clicking to close the preview, we set `imageSrc` to `undefined`.

Reply:
If it can't be any kind of falsy values except `undefined` why we couldn't use
`open={imageSrc}` or `open={!!imageSrc}`